src/rpc/governance.cpp

```diff
@@ -133,12 +133,14 @@ void gobject_prepare_help()
                 "3. time          (numeric, required) time this object was created\n"
                 "4. data-hex      (string, required)  data in hex string form\n"
                 "5. use-IS        (boolean, optional, default=false) InstantSend lock the collateral, only requiring one chain confirmation\n"
+                "6. outputHash    (string, optional) the single output to submit the fee from\n"
```

Comment:
Is it really for the "fee" or for the "collateral"?

Reply:
I've heard it referred to as fee, collateral gives the thought it can be taken back out

Reply:
"proposal fee" maybe?

Reply:
```suggestion
                "6. outputHash    (string, optional) the single output to submit the proposal fee from\n"
```